### ISSUE TITLE
Other languages support

### ISSUE THREAD

**Issue by macstalker** [joeblurton/doccounter#1]
It shows very weird words/lines/pages count on russian language. I think other non-roman languages affected too.

Thanks!

**Comment by joeblurton** [comment]
Yes, I there are specific issues with different charsets. Please see the specific project pages for each component. I can't offer an exhaustive list of compatibility, but if you find a fix then please do submit a pull request.

**Comment by macstalker** [comment]
ok, thanks!
